src/Tizen.NUI.Components/Controls/SelectGroup.cs

```diff
@@ -69,6 +69,13 @@ public abstract class SelectGroup
         [EditorBrowsable(EditorBrowsableState.Never)]
         public int SelectedIndex => selectedIndex;
 
+        /// <summary>
+        /// EnableMultiSelected is used to indicate if SelectGroup can select multiple SelectButtons.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
Please don't use `<since_tizen>` tag for EditorBrowsable items.

Reply:
Thank you for your  comment.
I thought that EditorBrowsable items are also required to have since_tizen tags because other EditorBrowsable items have since_tizen tags.
But now I see they are incorrect due to your comment.

I have removed the since_tizen tag.
Thank you!